packages/react-core/src/components/Card/Card.tsx

```diff
@@ -12,20 +12,18 @@ export interface CardProps extends React.HTMLProps<HTMLElement>, OUIAProps {
   className?: string;
   /** Sets the base component to render. defaults to article */
   component?: keyof JSX.IntrinsicElements;
-  /** Modifies the card to include hover styles on :hover */
+  /** @deprecated Modifies the card to include hover styles on :hover */
   isHoverable?: boolean;
-  /** @beta Modifies the card to include hoverable-raised styles on :hover, this styling is included by default with isSelectableRaised */
-  isHoverableRaised?: boolean;
   /** Modifies the card to include compact styling. Should not be used with isLarge. */
   isCompact?: boolean;
   /** Modifies the card to include selectable styling */
   isSelectable?: boolean;
```

Comment:
so, is this prop deprecated?

Reply:
Are we going to keep supporting the original plain variation of selectable cards?

Reply:
Yes. I was confused at first.  I thought the original variant prop also set the card to be selectable.